tests/purs/passing/PolyLabels.purs

```diff
@@ -4,7 +4,8 @@ import Prelude
 import Prim.Row
 import Effect
 import Effect.Console
-import Data.Symbol (class IsSymbol, SProxy(..), reflectSymbol)
+import Data.Reflectable (class Reflectable, reflectType)
+import Type.Proxy (Proxy(..))
```

Comment:
Deciding to replace `IsSymbol` with `Reflectable` was an ad-hoc decision. If we're going to do it, we should be consistent and do it everywhere. So, I'll revert this change and continue using `IsSymbol`.